code/modules/admin/watchlist.dm

```diff
@@ -36,6 +36,8 @@
 	if(browse)
 		watchlist_show(target_sql_ckey)
 
+	add_note(target_ckey, "Added to Watchlist - [reason]", null, usr.ckey, 0, null, 0)
```

Comment:
@ExcessiveUseOfCobblestone I got informed by an admin that I was watchlisted for "spamming OOC" or something along those lines by shadowlight, you can see the relevant complaint here
https://tgstation13.org/phpBB/viewtopic.php?f=23&t=8789